lib/components/getGroupMembershipForDN.js

```diff
@@ -21,12 +21,12 @@ let search
  * @param {function} callback The callback to invoke when done.
  * @constructor
  */
-function GroupMembersForDN (opts, dn, callback) {
+function GroupMembersForDN (opts, dn, callback, stack) {
   this.opts = opts
   this.dn = dn
   this.callback = callback
-
-  this.stack = new Map()
+  
```

Comment:
Please remove this extraneous whitespace. Running `npm run lint` should catch such issues.

Reply:
My eslint was not saying nothing but I fixed it so that npm run lint is happy aka silent.